script/view-surge/README.md

```
@@ -0,0 +1,462 @@
+      시딩은 bastion 에서 nohup 으로 돌아가므로 `script/aws/seed.sh --follow` 로 확인합니다.
+      규모를 줄여 측정했다면 결과 파일의 시드 규모 행을 실제 값으로 고칩니다.
+- [ ] 통계 갱신 — 대량 적재 직후에는 옵티마이저 통계가 낡아 EXPLAIN 의 rows 추정이 엉뚱합니다.
+      ```bash
+      mysql ... -e "ANALYZE TABLE post_view_count, view_surge_tracking;"
+      ```
+- [ ] 앱 기동 확인 — `curl -X POST "$BASE_URL/api/v4/posts/5999999/view-count"` 가 `"code":200`
+      (view-count 계열은 부하테스트용으로 비로그인 허용)
+- [ ] 서버 설정 확인 — `application.yml` 의 `view-surge:` 블록.
+      **본 측정은 기본값으로 합니다.** 로컬 스모크에서 낮춰 둔 값이 배포에 섞이지 않았는지 확인.
+
+      | 설정 | 기본값 | 의미 |
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)